eclass/bitcoincore.eclass

```diff
@@ -198,7 +198,14 @@ fi
 if [ "${BITCOINCORE_NEED_LIBSECP256K1}" = "1" ]; then
 	BITCOINCORE_COMMON_DEPEND="${BITCOINCORE_COMMON_DEPEND} $LIBSECP256K1_DEPEND"
 fi
-if [ "${PN}" != "libbitcoinconsensus" ] && ! use_if_iuse test; then
+if [ "${PN}" = "libbitcoinconsensus" ]; then
```

Comment:
Looks like you reverted the use_if_iuse and then applied again.  Can you squash these two.